### DISCUSSION TITLE
I lost my email and cannot recover my account

### DISCUSSION THREAD

**Discussion by chicofreitas** [community/community#53950] [New to GitHub]
Hi!

I'm in serious trouble here. I used LastPass to store my passwords. But on the last install of Ubuntu 23.04 I forgot my password on LastPass.

As a result, I lost access to several accounts, including my small business's github account.

How can I prove that the account is mine and regain access? The email I used was from a hosting service that no longer exists.

**Comment by neutrinomuon** [comment]
Hi @chicofreitas ,

I'm deeply sorry to hear about the situation you are in. Losing access to your LastPass password can be very stressful :(!

Firstly, I suggest you try to contact GitHub support **immediately**. While we, as users, can offer some guidance, GitHub support will be best equipped to help you regain access to your account.

Another option to regain access to your GitHub account, which perhaps you have already tried, is to try to recover your LastPass master password. LastPass has a password recovery that you can use to reset your master password. You can find the instructions for recovering your master password on the LastPass website. But, as I understand, the email no longer exists, correct? So, you are unable to recover your LastPass master password and this is the same for your GitHub account password... 

So, coming back, If the email address is no longer active, you should contact GitHub support and provide them with any information that can help them verify your identity. This may include details about your account such as the date it was created, the last time you accessed it, or any other relevant information that could help confirm your ownership of the account.

I hope this helps you,
